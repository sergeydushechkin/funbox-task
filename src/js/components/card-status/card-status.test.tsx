import * as React from "react";
import * as renderer from "react-test-renderer";

import CardStatus from "./card-status";

it(`Render CardStatus component`, () => {
  const cardStatusElement = renderer
    .create(
        <CardStatus
          isAvailable={false}
          title={`с курой`}
          description={`Филе из цыплят с трюфелями в бульоне.`}
          isActive={false}
          onClick={() => { /* noop*/ }}
        />
    )
    .toJSON();

  expect(cardStatusElement).toMatchSnapshot();
});
