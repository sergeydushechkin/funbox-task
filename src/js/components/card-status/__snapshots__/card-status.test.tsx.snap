// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CardStatus component 1`] = `
<p
  className="card__status card__status--not-available"
>
  Печалька, с курой закончился.
</p>
`;
